src/web/services/template-sessions.ts

```diff
@@ -0,0 +1,139 @@
+import { FeedbackParticipantType } from '../app/feedback-participant-type';
+import {
+  FeedbackQuestion,
+  FeedbackQuestionType,
+  NumberOfEntitiesToGiveFeedbackToSetting,
+} from '../app/feedback-question';
+import { FeedbackVisibilityType } from '../app/feedback-visibility';
+
+/**
+ * A template session.
+ */
+export interface TemplateSession {
+  name: string;
+  questions: FeedbackQuestion[];
+}
+
+/**
+ * Template sessions.
+ */
+export const templateSessions: TemplateSession[] = [
```

Comment:
This should be a JSON file instead? This is a form of data, not application code.

Reply:
I need `interface TemplateSession {}` defined somewhere. So I thought put them together would be more organised.

Reply:
More organized is one thing. The issue here is you're mixing up application code and application data (independent of logic).

Reply:
Ok. One problem of serialised it to JSON file is that we will loss reference to enum type (e.g. `FeedbackVisibilityType.RECIPIENT` will become `"RECIPIENT"`) Not sure whether it is a bit problem or not for project evolution.